### PR TITLE
Step one inspection

### PR DESCRIPTION
**Pull request by AnnaVel**
Addresses #3395

### CODE REVIEW

File: Rubberduck.Inspections/Concrete/StepIsNotSpecifiedInspection.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+using Rubberduck.Inspections.Results;
+using static Rubberduck.Parsing.Grammar.VBAParser;
+
+namespace Rubberduck.Inspections.Concrete
+{
+    public sealed class StepIsNotSpecifiedInspection : ParseTreeInspectionBase
+    {
+        public StepIsNotSpecifiedInspection(RubberduckParserState state) : base(state, CodeInspectionSeverity.Warning) { }
+
+        public override Type Type => typeof(StepIsNotSpecifiedInspection);
```
Comment (Vogel612) [inline Rubberduck.Inspections/Concrete/StepIsNotSpecifiedInspection.cs]: nothing to do with your PR, but we really wanted to get rid of this...

**Comment by retailcoder** [comment]
Nice! Seems there was a problem with merging upstream though.

File: Rubberduck.Parsing/Grammar/VBAParser.g4 [hunk 2]
```diff
@@ -486,6 +486,10 @@ selectEndValue : expression;
 
 setStmt : SET whiteSpace lExpression whiteSpace? EQ whiteSpace? expression;
 
+stepStmt :
+	STEP whiteSpace expression
+;
```
Comment (Vogel612) [inline Rubberduck.Parsing/Grammar/VBAParser.g4]: I'll need to pull those changes over into the ANTLR Upgrade as well, remdinder to myself to do that

**Comment by AnnaVel** [comment]
That's my obviously unsuccessful attempt to update my branch from the original branch. Git and I don't get along so well.

File: Rubberduck.Inspections/Concrete/StepIsNotSpecifiedInspection.cs (hunk 1 above)
Comment (retailcoder) [reply Rubberduck.Inspections/Concrete/StepIsNotSpecifiedInspection.cs]: @Vogel612 It's deprecated already... and actually needs to be removed in order to build 😄

**Comment by rkapka** [comment]
Please take into account that if #3718 gets merged before this PR, you'll need to add this inspection's default setting to the `.setting` file.

**Pull request merged**